### PR TITLE
Add new sensor intersection properties

### PR DESCRIPTION
**Pull request by bagnell**

### CODE REVIEW

File: Schema/Extensions/AGI/ConicSensor.jsonschema [hunk 1]
```diff
@@ -81,6 +81,30 @@
         "portionToDisplay": {
             "$ref": "Extensions/AGI/SensorVolumePortionToDisplay.jsonschema",
             "description": "What part of the sensor should be displayed."
-        }
+        },
+		"environmentConstraint": {
```
Comment (pjcozzi) [inline Schema/Extensions/AGI/ConicSensor.jsonschema]: Here and throughout below, are tabs used instead of spaces?

**Comment by pjcozzi** [comment]
Just that one comment.

@shunter do you want to review and merge?  I am not very familiar with this code.

**Comment by shunter** [comment]
:+1:

**Comment by pjcozzi** [comment]
Thanks @shunter @bagnell!

**Pull request merged**